pkg/pgclient/client.go

```diff
@@ -48,6 +50,7 @@ func ParseFlags(cfg *Config) *Config {
 	flag.IntVar(&cfg.ReportInterval, "tput-report", 0, "interval in seconds at which throughput should be reported")
 	flag.Uint64Var(&cfg.LabelsCacheSize, "labels-cache-size", 10000, "maximum number of labels to cache")
 	flag.Uint64Var(&cfg.MetricsCacheSize, "metrics-cache-size", pgmodel.DefaultMetricCacheSize, "maximum number of metric names to cache")
+	flag.IntVar(&cfg.ConnectionsPerProc, "connections-per-proc", -1, "maximum number of database connections per go process")
```

Comment:
I would keep this parameter but add a second that is a total max connections to sort of act like a failsafe, like if I scale up my instance I don't want to swamp my db completely, so basically, I want to be able to say give me 4 connections per proc up to 100 for the database, so then I can, for instance, use an autoscaler.